### DISCUSSION TITLE
Cannot find file through IDE search function well.

### DISCUSSION THREAD

**Discussion by Tomtommytomtom** [posva/unplugin-vue-router#372] [Ideas]
Before switching to unplugin-vue-router i could easily press ctrl + p and find my Page by typing in the name of the route. Now the name is defined in the definePage macro, and i have a bunch of files called index.vue.

It would be nice if we could somehow put the route name inside of the filename, since @ is already taken, maybe we could do something like "path/to/my/route/index_MyRouteName.vue"?

**Comment by posva** [comment]
It's worth exploring what other frameworks have done for this. If your files are named `index.vue`, search by the name of the folder which is the route too, e.g. `/users/index.vue` -> cmd+p > <kbd>us i</kbd>

**Comment by posva** [comment]
This is solved with groups which allows to name files however you want `index.vue` -> `(anything-goes).vue`
